BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/AzureServiceBus/EventBusAzureServiceBus.cs

```diff
@@ -106,6 +104,11 @@ public async void Publish(IntegrationEvent @event)
         _subscriptionManager.AddSubscription<T, TH>();
     }
 
+    public void StartConsuming()
+    {
+        RegisterSubscriptionClientMessageHandlerAsync().GetAwaiter().GetResult();
```

Comment:
I think we don't actually need to await this, do we? It just finishes at some time in the future and starts consuming.

Reply:
Don't we have the same warning in the following line: https://github.com/nmshd/backbone/blob/f5dfb9a75c4dcdc974db106b685ee36a6a3b4908/BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/GoogleCloudPubSub/EventBusGoogleCloudPubSub.cs#L84 then?